recipes/waybackpy/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "waybackpy" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/waybackpy-{{ version }}.tar.gz
+  sha256: 264aa84a76365e8355ede96cb6cf02a1945295724ddfb7d4a43e6f3d8f97740a
+
+build:
+  entry_points:
+    - waybackpy = waybackpy.cli:main
+  noarch: python
```

Projects with `console_scripts` entry points are not `noarch` because deploying bash/batch scripts is not cross-platform. Setuptools does a different thing for Windows and Linux. Try removing noarch and see what happens.
```suggestion
  entry_points:
    - waybackpy = waybackpy.cli:main
```